static/app/utils/replays/hooks/useLogReplayDataLoaded.tsx

```diff
@@ -45,6 +45,23 @@ function useLogReplayDataLoaded({fetchError, fetching, projectSlug, replay}: Pro
       finished_at_delta: replay.timestampDeltas.finishedAtDelta,
       replay_id: replayRecord.id,
     });
+
+    // TODO(session-replay-show-hydration-errors): remove this analytic on GA
+    if (!organization.features.includes('session-replay-show-hydration-errors')) {
+      // @ts-expect-error the hydration frames are filtered when the flag is off
+      const allFrames = replay._sortedBreadcrumbFrames;
```

Comment:
This is meant to be a private field, so a) i'm surprised typescript isn't complaining b) can you call `getChapterFrames` instead

Reply:
getChapterFrames is filtering out the frames i want https://github.com/getsentry/sentry/blob/615d9f558639b13cae5f023d463aeee152bc2875/static/app/utils/replays/replayReader.tsx#L298

Reply:
oh right, the other feature.

why is TS not complaining though?

Reply:
ts-expect-error?

Reply:
what if the ReplayReader class had a public method 'has hydration crumb'

something like:
`replay._sortedBreadcrumbFrames.some(...);`